app/presenters/metrics_presenter.rb

```diff
@@ -36,6 +37,7 @@ def metric_groups
       metric_groups.reverse! if order == Metrics::Order::Descending
 
       if order_by == Metrics::OrderBy::Name.identifier
+        metric_groups = metric_groups.reverse
```

Comment:
I've steered clear of doing anything with ```@order ``` because of the template, I don't want to be setting the wrong radio button.